lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -2,15 +2,29 @@
 # export_tasks.rake is a debugging tool to extract tasks from the
 # current foreman instance.
 #
-# Run "foreman-rake export_tasks" to export tasks whcih are not listed
-# as successful.
-# To export all tasks "foreman-rake export_tasks tasks=all"
-#   to specify the number of days of tasks to gather:  days=60 (defaults to 60)
+# Run "foreman-rake foreman_tasks:export_tasks" to export tasks
+
+require 'csv'
 
 namespace :foreman_tasks do
-  desc 'Export Dynflow Tasks'
+  desc <<DESC
+Export dynflow tasks based on filter. ENV variables:
+
+  * TASK_SEARCH     : scoped search filter (example: 'label = "Actions::Foreman::Host::ImportFacts"')
+  * TASK_FILE       : file to export to (minus the file extension)
+  * TASK_FORMAT     : format to use for the export (either html or csv)
 
+If TASK_FILTER is not defined, it defaults to unsuccessful tasks newer than 60
+days ago. The default TASK_FORMAT is html which requires a tar.gz file extension.
+DESC
   task :export_tasks => :environment do
+    deprecated_options = {:tasks => "TASK_SEARCH",
+                          :days => "TASK_SEARCH",
```

Comment:
the days is still pretty useful option IMO, compared to users having to format the date one their own.